libs/coin-modules/coin-evm/src/__tests__/coin-tester/scenarios/ethereum.ts

```diff
@@ -0,0 +1,252 @@
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  subAccountId: encodeTokenAccountId(
+    "js:2:ethereum:0x3313797c7B45F34c56Bdedc0179992A4d435AF25",
+    USDC_ON_ETHEREUM,
+  ),
+  expect: account => {
+    const { operations } = account;
+    const sendOperation = operations.find(operation => operation.transactionSequenceNumber === 1);
+    expect(sendOperation?.transactionRaw?.amount).toBe("80");
+  },
+};
+
+const scenarioERC721Transaction: ScenarioTransaction<EvmTransaction & EvmNftTransaction> = {
+  name: "Send ERC721",
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  mode: "erc721",
+  nft: {
+    tokenId: "3368",
+    contract: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+    quantity: new BigNumber(1),
+    collectionName: "Bored Ape",
+  },
+  expect: account => {
+    const sendOperation = account.operations.find(
+      operation => operation.transactionSequenceNumber === 2,
+    );
+
+    expect(
+      sendOperation?.nftOperations?.find(
+        op => op.contract === "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+      ),
+    ).toBeDefined();
+  },
+};
+
+const scenarioERC1155Transaction: ScenarioTransaction<EvmTransaction & EvmNftTransaction> = {
+  name: "Send ERC1155",
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  mode: "erc1155",
+  nft: {
+    tokenId: "951",
+    collectionName: "Clone X",
+    contract: "0x348FC118bcC65a92dC033A951aF153d14D945312", // EIP55 checksum of 0x348fc118bcc65a92dc033a951af153d14d945312
+    quantity: new BigNumber(2),
+  },
+  expect: account => {
+    const sendOperation = account.operations.find(
+      operation => operation.transactionSequenceNumber === 3,
+    );
+    expect(
+      sendOperation?.nftOperations?.find(
+        op => op.contract === "0x348FC118bcC65a92dC033A951aF153d14D945312",
+      ),
+    ).toBeDefined();
+  },
+};
+
+const initUSDCAccount = async (provider: ethers.providers.JsonRpcProvider, address: string) => {
+  // USDC
+  const addressToImpersonateBinance = "0x47ac0Fb4F2D84898e4D9E7b4DaB3C24507a6D503"; // Binance
+  await provider.send("anvil_impersonateAccount", [addressToImpersonateBinance]);
+  const sendUSDC = {
+    from: addressToImpersonateBinance,
+    to: USDC_ON_ETHEREUM.contractAddress,
+    data: ERC20Interface.encodeFunctionData("transfer", [
+      address,
+      ethers.utils.parseUnits("100", USDC_ON_ETHEREUM.units[0].magnitude),
+    ]),
+    value: ethers.BigNumber.from(0).toHexString(),
+    gas: ethers.BigNumber.from(1_000_000).toHexString(),
+    type: "0x0",
+    gasPrice: (await provider.getGasPrice()).toHexString(),
+    nonce: "0x" + (await provider.getTransactionCount(addressToImpersonateBinance)).toString(16),
+    chainId: "0x" + (await provider.getNetwork()).chainId.toString(16),
+  };
+
+  const hash = await provider.send("eth_sendTransaction", [sendUSDC]);
+  await provider.send("anvil_stopImpersonatingAccount", [addressToImpersonateBinance]);
+  await provider.waitForTransaction(hash);
+};
```

Comment:
Just a comment for later, but rather than having those unique methods for each init, I guess we can create a helper to "impersonnate" an account and just execute a random ethers.Transaction in its name, because we're doing the same thing every time here 🤷‍♂️